pkgs/top-level/all-packages.nix

```
@@ -10607,6 +10607,8 @@ in
 
   clearsilver = callPackage ../development/libraries/clearsilver { };
 
+  clFFT = callPackage ../development/libraries/clFFT { };
```

```suggestion
  clfft = callPackage ../development/libraries/clfft { };
```
As per manual guidelines https://nixos.org/nixpkgs/manual/#sec-package-naming and https://nixos.org/nixpkgs/manual/#sec-organisation